demo/client/src/view.ts

```diff
@@ -174,6 +175,10 @@ export abstract class DemoView extends PathfinderView implements RenderContext {
     protected colorBufferHalfFloatExt: any;
 
     private wantsScreenshot: boolean;
+    private vrDisplay: VRDisplay | null;
```

Comment:
nit: can you put a newline above this? Let's keep all the VR-specific stuff together.